druid-shell/src/platform/windows/window.rs

```diff
@@ -1659,7 +1659,51 @@ impl WindowHandle {
     }
 
     pub fn content_insets(&self) -> Insets {
-        warn!("WindowHandle::content_insets unimplemented for windows.");
+        if let Some(w) = self.state.upgrade() {
+            let hwnd = w.hwnd.get();
+            unsafe {
+                let mut info = WINDOWINFO {
+                    cbSize: mem::size_of::<WINDOWINFO>() as u32,
+                    rcWindow: RECT {
+                        left: 0,
+                        top: 0,
+                        right: 0,
+                        bottom: 0,
+                    },
+                    rcClient: RECT {
+                        left: 0,
+                        top: 0,
+                        right: 0,
+                        bottom: 0,
+                    },
+                    dwStyle: 0,
+                    dwExStyle: 0,
+                    dwWindowStatus: 0,
+                    cxWindowBorders: 0,
+                    cyWindowBorders: 0,
+                    atomWindowType: 0,
+                    wCreatorVersion: 0,
+                };
```

Comment:
You could write this all as,

```rust
let mut info: WINDOWINFO = mem::zeroed();
info.cbSize = mem::size_of::<WINDOWINFO>() as u32;
```

Reply:
Yes indeed that's better ! I've done the changes.